portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java

```diff
@@ -98,6 +106,37 @@ public void testGetWithDefaultValue() {
 			_TEST_VALUE, SystemProperties.get(_TEST_KEY, "defaultValue"));
 	}
 
+	@Test
+	public void testLoad() throws Exception {
+		String userDir = StringUtil.replace(
+			System.getProperty("user.dir"), CharPool.BACK_SLASH,
+			CharPool.FORWARD_SLASH);
+
+		File systemPropertiesFile = new File(
+			userDir + "/portal-kernel/test-classes/unit/system.properties");
+
+		try {
+			systemPropertiesFile.createNewFile();
+
+			try (FileWriter fileWriter = new FileWriter(systemPropertiesFile)) {
+				fileWriter.write(
+					StringBundler.concat(
+						"#test case", StringPool.NEW_LINE, _TEST_KEY, "=\\",
+						StringPool.NEW_LINE, "\\", StringPool.NEW_LINE, "#",
+						StringPool.NEW_LINE, _TEST_VALUE, StringPool.NEW_LINE));
+
+				fileWriter.flush();
+			}
+
+			SystemProperties.load(SystemPropertiesTest.class.getClassLoader());
+
+			Assert.assertEquals(_TEST_VALUE, SystemProperties.get(_TEST_KEY));
+		}
+		finally {
+			systemPropertiesFile.delete();
+		}
+	}
```

Comment:
@W2jx Here you are test two parts of logic together, one is find the right file based on class loader, the second is to parse comments. While, here, you should focus only on the second part because this is what you changed in your previous commit.
Please write a test method to test the _load method.
Also, you can change your _load method to accept an inputstream instead of URL, this way, you do not have to create a real file.

Tina